lms/static/js/student_profile/views/learner_profile_fields.js

```diff
@@ -16,22 +17,29 @@
             },
 
             showNotificationMessage: function () {
-                var accountSettingsLink = '<a href="' + this.options.accountSettingsPageUrl + '">' + gettext('Account Settings page.') + '</a>';
+                var accountSettingsLink = StringUtils.interpolate('<a href="{url}">Account Settings page.</a>',
```

Comment:
@andy-armstrong @kevinjkim: I think you need to wrap the opening `<a>` tag with an `HtmlUtils.HTML()` call as well, otherwise it will be escaped.

Something like:
```
accountSettingsLinkHtml = HtmlUtils.joinHtml(
    HtmlUtils.interpolateHtml(
        HtmlUtils.HTML('<a href="{settings_url}">'), {settings_url: accountSettingsPageUrl}
    ),
    gettext('Account Settings page.'),
    HtmlUtils.HTML('</a>')
);
```